dev-util/hxtools/patches/hxtools-20180301.patchset

```diff
@@ -131,5 +97,49 @@ index 2c973a1..88d3a73 100644
 -xcp_LDADD     = ${libHX_LIBS}
 +#xcp_LDADD     = ${libHX_LIBS}
 -- 
-2.7.0
+2.16.4
+
+
+From 809a9f9f08af7485402f2e270621f8a06f46c22a Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Thu, 16 Aug 2018 11:01:34 +0200
+Subject: Use correct path to perl
+
+
+diff --git a/smm/aumeta b/smm/aumeta
+old mode 100755
+new mode 100644
+index 2b6959d..af76eac
+--- a/smm/aumeta
++++ b/smm/aumeta
+@@ -1,4 +1,4 @@
+-#!/usr/bin/perl
++#!/bin/env perl
```

Comment:
The more "universal" path should be `/usr/bin/env perl`. It's supported by us and some *nix out there with weird `PATH` as well

Reply:
Bumped :)